packages/commonwealth/server/models/reaction.ts

```diff
@@ -153,16 +152,15 @@ export default (
           ],
           unique: true,
         },
-        { fields: ['chain', 'thread_id'] },
-        { fields: ['chain', 'comment_id'] },
-        { fields: ['canvas_hash'] },
```

Comment:
Any reason for removing this index?

Reply:
It doesn't exist in the DB (there was no migration to add it) so it's either I remove it from the model or add it via migration. I chose not to add it to the db since it is only used by the query in the `getCanvasData` function which itself is only used by the `/oplog` route. As far as I can tell this route is very rarely used (if ever) so I think it's better to optimize for INSERTs by not adding the index.